### ISSUE TITLE
Possible solution for `MIDIPacketNext` `.pointee` Swift crashes

### ISSUE THREAD

**Issue by orchetect** [krevis/MIDIApps#86]
Disregard - issue persists and remains elusive after further testing.

**Comment by krevis** [comment]
Thanks for the information in any case! I wasn’t aware of `

**Comment by orchetect** [comment]
The issue is brutally elusive so I'm taking a hint from your implementation to see if a C-based MIDIPacketNext wrapper and unsafe memory access can prevent crashes.

This has been a long-standing issue with CoreMIDI in Swift and I really wish Apple would care enough to fix it.

I've seen many other libraries dogged with this issue because it doesn't always crop up right away.

**Comment by orchetect** [comment]
Yes `unsafeSequence()` and the event iterator work well but they are only available on more recent Apple platforms - macOS 10.15, iOS 13.0, macCatalyst 13.0.

What I found was different memory issues and crashes would appear depending on the OS version, so I had to add some robustness to handling the pointers on older platforms. If you're curious I can link some of the solutions. But If you're comfortable dropping support for older platforms then just go with the new API.
